std_msgs/Header header

float64 centre_radius
float64 centre_offset